test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/Dnu.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dnu
+    {
+        private readonly string _sdkPath;
```

Comment:
I think all the command methods in this class need extra arguments just like `Publish`